test/amqp-integration-test.py

```diff
@@ -222,14 +223,34 @@ def cleanup():
     else:
         print("\n\nFAILURE %d" % exit_status)
 
-
 exit_status = ExitStatus.OK
 tempdir = tempfile.mkdtemp()
+
+parser = argparse.ArgumentParser(description='Run integration tests')
+parser.add_argument('--all', dest="run_all", action="store_true",
+                    help="run all of the clients' integration tests")
+parser.add_argument('--letsencrypt', dest='run_letsencrypt', action='store_true',
+                    help="do not run the letsencrypt's (the python client's) integration tests")
```

Comment:
You seem to have the help for this and the argument below mixed up? (`--node` _will_ 'run the node clien'ts integration tests').